.dockerignore

```diff
@@ -0,0 +1,12 @@
+.env
+.env.template
+.env.test
+.gitignore
+geckodriver.exe
+geckodriver.log
+README.md
+.github
+.idea
+.pytest_cache
+.venv
+.vscode
```

Comment:
You got the most important things, and it's not that important since you are careful with what you COPY in anyway.

But some other things you could include are the `__pycache__` folders and `.git`